---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a Bioinformatican by eduction. I develop systems biology models and machine learning algorithms to analyse the complexities of biological systems at the single-cell level. Currently, I am developing models to enhance the functional interpretation of single cell RNA-seq (scRNA-seq) data by integrating genetics and single cell ATAC-Seq (scATAC-seq) data in the context of different human brain disorders. 



## Short CV
I  received a master  in Bioinformatics from the [Max Planck Institute for Informatics](https://www.mpi-inf.mpg.de/home/) and Saarland University, Germany. I did my PhD. with the [International Max Planck Research School for Computer Science](https://www.imprs-cs.de/) at the Max Planck Institute for Informatics. Since April 2019, I am a postdoctoral research associate with a joint appointment at Broad Institute of MIT and Harvard, and MIT Computer Science and Artificial Intelligence Laboratory (CSAIL).



## Summary

The cell is a highly mysterious intelligent machine that fascinates my imagination and arouses my curiosity. My
research revolves around development and application of principled machine learning methods for interpreting
complex biological data sets. I love to design novel machine learning algorithms, especially when they allow the
analysis of data in ways that have not yet been possible. I am interested in working on fundamental biological
problems that would have an impact on our understanding of human cells and disease mechanisms. An ongoing theme in my research is the integration of multi-omics data sets to understand underlying biological processes that contribute to a specific phenotype. In particular, I work on methods to integrate and analyze high-throuput omics data.

## Single cell genomics

Single-cell multi-omics data provides us an opportunity to understand biological processes at the fundamental
level. These technologies drive new discoveries and are rapidly advancing, which calls for the constant design
and improvement of computational methods for integration, interpretation, and analysis. I would like to
develop new methods for single cell data analysis using a combination of rigorous statistical approaches. These
methods are necessary to cope with the inherent noise of these technologies and to focus on computationally
efficient design and implementation such that resulting method can handle thousands of single cell omics data.
## Computational Epigenomics

I would like to continue my work on understanding the system perspective of transcriptional and post-transcriptional gene regulation, answering questions like: What are the rules of enhancer gene targeting? How do structural variations in enhancers effect target gene regulation? Which epigenetic signatures contribute to establishing cell
identity? Which regulatory interactions are involved in changing cell identity?