---
layout: archive
permalink: /year-archive/
title: " "
author_profile: true
redirect_from:
  - /wordpress/blog-posts/
---
<h1>Summary</h1>
 <br/><img src='/images/Azim_ResearchSummaryFinal.png'>



<p>The cell is a highly mysterious intelligent machine that fascinates my imagination and arouses my curiosity. My
research revolves around development and application of principled machine learning methods for interpreting
complex biological data sets. I love to design novel machine learning algorithms, especially when they allow the
analysis of data in ways that have not yet been possible. I am interested in working on fundamental biological
problems that would have an impact on our understanding of human cells and disease mechanisms. An ongoing theme in my research is the integration of single cell multi-omics data sets to understand underlying biological processes that contribute to a specific phenotype. </p>



<h1>Projects</h1>


<h2>IsoRegNet: A multi-task learning framework to model and analyze transcript isoform regulatory network</h2>

 <br/><img src='/images/IsoregNetfig.png'>

<b2>Understanding gene regulation is an important aspect in the context of
cell differentiation, disease and environmental changes. Due to the complexity
of gene regulation, methods for the inference of gene regulatory
networks (GRNs), are often limited to one type of regulator, for example
only DNA binding transcription factors or microRNAs. Also the diversity
in gene isoforms is rarely taken into account during the construction of
GRNs. I've developed a novel principled approach that allows to learn associations
between transcriptional and post-transcriptional regulators with
genes or their transcript isoforms. Using a novel multi-task formulation
we design a regression framework, which allows to overcome some of the
noise in isoform expression level estimates, leading to accurate predictions.<b2>

<h2>Inferring genome-wide networks of competing endogenous RNAs</h2>
 <br/><img src='/images/SpongFig.png'>

  MicroRNAs (miRNAs) are important non-coding post-transcriptional regulators that are involved in many biological processes and human diseases.
  Individual miRNAs may regulate hundreds of genes, giving rise to a complex gene regulatory network in which transcripts carrying 
  miRNA binding sites act as <strong>competing endogenous RNAs (ceRNAs)<strong>. Several methods for the analysis of ceRNA interactions exist,
  but these do often not adjust for statistical confounders or address the problem that more than one miRNA interacts with a target transcript.

  
  We developed <strong>SPONGE<strong>, a method for the fast construction of ceRNA networks. SPONGE uses <strong>multiple sensitivity correlation<strong>, a newly defined measure for which we can estimate a distribution under a null hypothesis. SPONGE can accurately quantify the contribution of multiple miRNAs to a ceRNA interaction with a probabilistic model that addresses previously neglected confounding factors and allows fast P-value calculation, thus outperforming existing approaches. We applied SPONGE to paired miRNA and gene expression data from The Cancer Genome Atlas for studying global effects of miRNA-mediated cross-talk.
  Our results highlight already established and novel protein-coding and non-coding ceRNAs which could serve as biomarkers in cancer.
  
<h1>Single cell genomics</h1>

<p>Single-cell multi-omics data provides us an opportunity to understand biological processes at the fundamental
level. These technologies drive new discoveries and are rapidly advancing, which calls for the constant design
and improvement of computational methods for integration, interpretation, and analysis. I
develop new methods for single cell data analysis using a combination of rigorous statistical approaches. These
methods are necessary to cope with the inherent noise of these technologies and to focus on computationally
efficient design and implementation such that resulting method can handle thousands of single cell omics data.</p>





<h1>Computational Epigenomics</h1>
<p>I would like to continue my work on understanding the system perspective of transcriptional and post-transcriptional gene regulation, answering questions like: What are the rules of enhancer gene targeting? How do structural variations in enhancers effect target gene regulation? Which epigenetic signatures contribute to establishing cell
identity? Which regulatory interactions are involved in changing cell identity?</p>





{% include base_path %}
{% capture written_year %}'None'{% endcapture %}
{% for post in site.posts %}
  {% capture year %}{{ post.date | date: '%Y' }}{% endcapture %}
  {% if year != written_year %}
    <h2 id="{{ year | slugify }}" class="archive__subtitle">{{ year }}</h2>
    {% capture written_year %}{{ year }}{% endcapture %}
  {% endif %}
  {% include archive-single.html %}
{% endfor %}
